src/content/documentation/develop/developing-extensions-for-firefox-for-android.md

```diff
@@ -39,8 +41,43 @@ However, instructions are provided for the steps you need to take should you cho
 
 {% capture content_with_toc %}
 
+## Testing instructions for Firefox for Android 79+ (Fenix)
+:::note alert
+Persistent loading is currently not supported. These instructions are for temporary loading.
+:::
+
+- Install or update [web-ext](https://github.com/mozilla/web-ext) to version 4.1.0 or later.
+
+- Install or update to the latest version of [Firefox Nightly for Android](https://play.google.com/store/apps/details?id=org.mozilla.fenix).
+
+- Make sure you have adb installed ([Linux](https://dl.google.com/android/repository/platform-tools-latest-linux.zip), [Mac](https://dl.google.com/android/repository/platform-tools-latest-darwin.zip), [Windows](https://dl.google.com/android/repository/platform-tools-latest-windows.zip)) and in your PATH.
+
+- Connect your Android device via USB.
+
+- Make sure you have [Android's USB debugging](https://developer.android.com/studio/debug/dev-options) enabled. You can run `adb devices` to verify.
+
+- In the settings for Firefox Nightly, enable "Remote debugging via USB."
```

Comment:
Nit, it happened at least once that the settings behind "Remote debugging via USB" was cleared across a Nightly update, it may be reasonable to rephrase this to something similar to the previous bullet point, something like `Make sure you have "Remove debugging via USB" enabled in the Firefox Nightly for Android`.

Maybe we should specify also that "Remote debugging via USB" is in the Firefox for Android Settings view?

Reply:
I *think* this incoming edit addresses this nit? If not I might need more direction. 😅